README.md

```diff
@@ -29,17 +29,30 @@ Code::Blocks SVN rev 9843, because there are some changes in the Scripted wizard
 ## Install
 
 Certain files should be placed or symlinked inside the User data folder of Code::Blocks.
-On Linux this is /home/$USER/.local/share/codeblocks
+On Linux this is /home/$USER/.local/share/codeblocks and ~/.local/share/codeblocks
 
 1. You need to install AVR compiler (AVR GCC) and set it up inside Code::Blocks
-2. You should create a symlink for the wizard. Link the root of this repo to ~/.local/share/codeblocks/templates/wizard/arduino
+2. On Linux, you should create a symlink for the wizard. Link the root of this repo to ~/.local/share/codeblocks/templates/wizard/arduino
```

Comment:
This line looks really bad when renderer. Probably it should be something like - Place the wizard where codeblocks can find it. 2.1 On linux do ... 2.2. on windows do this...
Or probably best if the windows and linux instructions are totally separated. No one will install them both at the same time.